typescript/infra/src/govern/HyperlaneCoreGovernor.ts

```diff
@@ -29,10 +29,10 @@ export class HyperlaneCoreGovernor extends HyperlaneAppGovernor<
       case MailboxViolationType.DefaultIsm: {
         let ismAddress: string;
         if (typeof violation.expected === 'object') {
-          const ism = await this.checker.ismFactory.deploy(
-            violation.chain,
-            violation.expected,
-          );
+          const ism = await this.checker.ismFactory.deploy({
+            chain: violation.chain,
+            config: violation.expected,
+          });
```

Comment:
is this still needed?

Reply:
Why not? Also, I just updated the params structure.